app/controllers/compute_resources_controller.rb

```diff
@@ -36,6 +36,10 @@ def create
       @compute_resource.valid?
       process_error
     end
+  rescue StandardError => e
+    process_error(
+      error_msg: _('Error while trying to create resource: %s' % e.message),
```

Comment:
i think the `% e.message` should go outside the `_()` translation block. Also it would be probably nice to log the full backtrace of the exception somewhere in case we would need to debug it.